cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -37,6 +37,24 @@ def create_ticket(self, ticket: TicketCreate, attachments: list[Path] = None) ->
         response.raise_for_status()
         return TicketResponse.model_validate(response.json())
 
+    def _convert_ticket_to_multipart_data(
```

Comment:
This logic does not belong in the `FreshdeskClient`. The client should be really stupid, just sending requests and returning responses parsed into models.

I suggest moving it into the service wrapping around the client, the ticket handler and then passing the correctly formatted data into the client.

Reply:
Additionally, assuming that the `_convert_ticket_to_multipart_data` belongs here in the client, it does not require any state from the class and as such, it just bloats the client class with unnecessary implementation details. It should be extracted to a standalone function to reduce clutter in the class.

This point is moot given that the logic does not belong in the client.